scapy/pton_ntop.py

```diff
@@ -77,25 +77,53 @@ def inet_ntop(af, addr):
         except AttributeError:
             pass
 
-        # IPv6 addresses have 128bits (16 bytes)
-        if len(addr) != 16:
+        return _ipv6_bin_to_str(addr)
+    else:
+        raise Exception("Address family not supported yet")
+
+
+def _ipv6_bin_to_str(addr):
+    # IPv6 addresses have 128bits (16 bytes)
+    if len(addr) != 16:
+        raise Exception("Illegal syntax for IP address")
```

Comment:
Since the address is in binary form, maybe `"Illegal IPv6 address"` would be better?
Or even better, we should mimic the behavior of `socket.inet_ntop`:
```Python
        raise ValueError("invalid length of packed IP address string")
```